File: 프로그래머스/0/181871. 문자열이 몇 번 등장하는지 세기/문자열이 몇 번 등장하는지 세기.cs
using System;

public class Solution {
    public int solution(string myString, string pat) {
        int answer = 0;
        for(int i = 0 ; i < myString.Length - pat.Length +1;i++)
        {
            string sub = myString.Substring(i,pat.Length);
            if(sub == pat)
                answer++;
        }
        return answer;
    }
}

File: 프로그래머스/0/181871. 문자열이 몇 번 등장하는지 세기/문자열이 몇 번 등장하는지 세기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string myString, string pat) {
    int answer = 0;
    for(int i = 0; i < myString.size() - pat.size() + 1; i++)
    {
        string temp = myString.substr(i, pat.size());
        if(temp == pat)
            answer++;
    }
    return answer;
}